package.json

```diff
@@ -50,7 +50,7 @@
   "homepage": "https://github.com/SU-SWS/decanter-react#readme",
   "dependencies": {
     "cnbuilder": "^2.6.0",
-    "decanter": "github:SU-SWS/decanter#v7",
+    "decanter": "github:SU-SWS/decanter#7.0.0-alpha.2",
```

Comment:
```suggestion
    "decanter": "^7.0.0-alpha.2",
```

Reply:
`npm i decanter@^7.0.0-alpha.2`

This should then pull the package through the npm package repository vs directly through ssh+github.